helm/cluster-autoscaler-app/values.yaml

```diff
@@ -8,17 +8,32 @@ groupID: 65534
 port: 8085
 
 # Make sure to update the docs at https://github.com/giantswarm/docs/blob/master/src/layouts/shortcodes/autoscaler_utilization_threshold.html if you change the defaults below
-configmap:
-  expander: "least-waste"
-  scanInterval: "10s"
-  scaleDownUtilizationThreshold: 0.65
-  skipNodesWithLocalStorage: "false"
-  skipNodesWithSystemPods: "true"
-
 image:
   registry: quay.io
   name: giantswarm/cluster-autoscaler
   tag: v1.15.2
 
-clusterID:
-  "test-cluster"
+clusterID: "test-cluster"
+provider: "changeme"
+
+azure:
+  subscriptionID: "changeme"
+  managedIdentity: true
+  vmType: VMSS
+
+extraArgs:
+  scan-interval: 10s
+  scale-down-utilization-threshold: 0.65
+  skip-nodes-with-system-pods: true
+  skip-nodes-with-local-storage: false
+  expander: least-waste
+  # write-status-configmap: true
```

Comment:
Why are these commented out? Is it so the user can enable them?

I'm not sure this is the right approach. I think it could be documented in the README instead.